.github/workflows/ci.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           path: |
             ~/.cache/pip
```

Comment:
Would it make sense to have separate caches by (python, airflow)?

Reply:
If I'm understanding your comment correctly, we do maintain separate caches based on the (python version)-(airflow version) combination with the below `key` attribute.

Did you mean something else?